components/wizard/structure/stepper.md

```diff
@@ -0,0 +1,366 @@
+---
+title: Stepper
+page_title: Wizard Stepper
+description: Stepper of the Wizard for Blazor.
+slug: wizard-structure-stepper
+tags: telerik,blazor,wizard,stepper
+published: True
+position: 0
+---
+
+# Wizard Stepper
+
+One of the main elements of the Wizard component is Stepper. The Wizard component utilizes the [Stepper]({%slug stepper-overview%}) component internally.
```

Comment:
is **a**/**the** a stepper. **The Wizard component utilizes the Telerik UI for Blazor Stepper** maybe sounds better.

Reply:
Done